components/icon-button.jsx

```diff
@@ -4,18 +4,23 @@ var Link = Router.Link;
 var ImageTag = require('./imagetag.jsx');
 
 var LinkSwap = React.createClass({
+  propTypes: {
+    linkTo: React.PropTypes.string,
+    href: React.PropTypes.string,
+    onClick: React.PropTypes.func
```

Comment:
Perhaps it might be useful to just pass `{...this.props}`, or `{..._.omit(this.props, 'linkTo', 'href')}` into the ultimate `Link`/``a` we render? That way we can pass any props that are valid for `<a>` tags (I believe `<Link>` already passes through props itself to the underlying `<a>` it renders).